.eslintrc.js

```diff
@@ -47,6 +47,12 @@ module.exports = {
         'jest/expect-expect': 'off',
       },
     },
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      rules: {
+        'no-undef': 'off',
```

Comment:
Is there an alternative fix for this than turning off the lint rule (or a devex task to address this later)? It almost seems counterintuitive to use TS but turn off this rule!!

Reply:
<img width="482" alt="Screen Shot 2022-09-19 at 5 23 34 PM" src="https://user-images.githubusercontent.com/1868782/191120706-d4be3acc-18bb-409c-bf36-39c7e4943d72.png">

so typescript will actually give a different warning if you try to invoke a function or use something that was not defined, so we don't even need eslint to catch these types of errors since typescript will find it for us.

Reply:
Ooh nice!! TY for showing me